libweb3jsonrpc/Test.cpp

```diff
@@ -125,7 +177,15 @@ std::string Test::test_importRawBlock(string const& _blockRLP)
     catch (ImportBlockFailed const& e)
     {
         cwarn << diagnostic_information(e);
-        throw JsonRpcException("Block import failed.");
+        auto importResult = boost::get_error_info<errinfo_importResult>(e);
+        std::string detailedError = "empty errorInfo_inportResult";
+        if (importResult)
```

Comment:
You don't really need this check, so just check for the nested exception only

Reply:
Or you could additionaly check import result and handle its values, too https://github.com/ethereum/aleth/blob/255a75ec4fc8ed7fce3449404595d7589d5d907f/libethcore/Common.h#L95-L107

with something like
```
if (auto nested = boost::get_error_info<errinfo_nestedException>(e))
    detailedError = exceptionToErrorMessage(*nested);
else if (auto importResult = boost::get_error_info<errinfo_importResult>(e))
    detailedError = importResultToErrorMessage(*importResult);
```